lib/raster/get_row.c

```diff
@@ -368,6 +372,9 @@ static void gdal_values_int(int fd, const unsigned char *data,
         case GDT_Byte:
             c[i] = *(GByte *)d;
             break;
+        case GDT_Int8:
+            c[i] = *(GInt16 *)d;
```

Comment:
I'm not familiar at all with that code, but I believe this should be:
```suggestion
            c[i] = *(signed char *)d;
```

Reply:
Included in bc9f6a89890e882a19d8a0e35fb92346d8183c78

Reply:
Why not `c[i] = *(GInt8 *)d;` to use the GDAL API and for consistency in the file?

Reply:
yes, although I'm not  big fan of our custom type . Using C99's int8_t would probably be better

Reply:
Indeed, standard fixed sized integers for fixed sized data types are certainly better.